Jenkinsfile

```diff
@@ -0,0 +1,26 @@
+#!groovy
```

Comment:
NIT: Pretty sure that in general Jesse does not like it ^. Maybe because, and I tend to agree, we should encourage people to think this is actual Groovy.